mods/cnc/rules/aircraft.yaml

```diff
@@ -1,13 +1,15 @@
 TRAN:
 	Inherits: ^Helicopter
 	Valued:
-		Cost: 750
+		# Cost: 750
```

Comment:
Please don't out-comment things, just replace them. We keep the full history of changes via git blame.